test/ex01/template.js

```diff
@@ -0,0 +1,113 @@
+/*
+ * DELETE ME TEXT: Make a copy, do not modify.
+ * Implements Excercise 01: Fun with Firestore SDK.
+ * @Author: Your Name
+ * @Date: 2021/01/DD
+ */
+import firebase, { firestore } from "firebase/app";
+import "firebase/firestore";
+
+import {
+  TOP_LEVEL_COLLECTION_NAME,
+  TEST_DOCUMENT_NAME,
+  COMPLETED_STEPS_DOC_NAME,
+  RESEARCH_ITEMS_COLLECTION_NAME,
+} from "./constants";
+
+/**
+ * Creates a sub collection in the given top level collection.
+ *
+ * The create sub collection stores documents that contain info about research
+ * documents we've read. Also updates the steps completed collection to reflect
+ * progress.
+ *
+ * @param {firestore.CollectionReference} topLevelCollection The reference to
+ * the top level collection.
+ * @param {firestore.CollectionReference} stepsCompletedCollection The
+ * reference to the collection used to track progress.
+ */
+function createResearchItemsCollection(
+  topLevelCollection,
+  stepsCompletedCollection
+) {
+  throw "NotImplementedError: Implement Me!";
+  // 1. Create a collection inside the top level collection.
+
+  // 2. Update the steps completed collection
+}
+
+/**
+ * Adds the required research items into the collection.
+ *
+ * Also updates the steps completed collection to refelect progress.
+ *
+ * @param {firestore.CollectionReference} researchItemsCollection The reference
+ * to the collection where research items will be added.
+ * @param {firestore.CollectionReference} stepsCompletedCollection The
+ * reference to the collection used to track progress.
+ */
+function allAllResearchItems(
+  researchItemsCollection,
+  stepsCompletedCollection
+) {
+  throw "NotImplementedError: Implement Me!";
+
+  console.log(
```

Comment:
Maybe we/I can add a print function that they can call to print out the created data/collections while they're implementing so they get a visual